symfony/psr-http-message-bridge/2.2/config/packages/psr_http_message_bridge.yaml

```diff
@@ -0,0 +1,11 @@
+psr_http_message:
+
+    # PSR-17 factories using the selected implementation (nyholm or diactoros)
+    # This functionality is enabled by default if nyholm/psr7 is available.
+    #message_factories:
+    #    enabled: true
+    #    implementation: nyholm
+
+    # Message converters that allow controllers to operate on PSR-7 requests/responses instead of HttpFoundation
```

Comment:
not "instead of" - we can remove the end of the sentence

Reply:
Agreed, changed.